starter-code/css/main.css

```diff
@@ -6,3 +6,79 @@ Green: #00B172
 White: #FFF
 
 */
+
+
+navbar img{
+    float:left;
+    width:18%;
+    height:auto;
+}
+
+navbar ul{
+    font-size:22px;
+    float:right;
+    list-style-type: none;
+    margin-right:50px;
+
+}
+navbar ul > li{
+    display:inline;
+}
+
+navbar{
+    position: fixed;
+    background-color: #FFF;
+    margin-top:0;
+
+}
+navbar a{
+    text-decoration:none;
+    color:black;
+    margin-left:15px;
+
+}
+
+.large-image-with-background-text{
+    background: url("../images/landing.jpg") no-repeat center 100px ;
+    background-size:cover;
+    padding-top:70%;
+    margin: 0;
+   
+    
+}
+.large-image-with-background-text p{
+        color:white;
+        position: absolute;
+        top: 85%;
+        left: 52%;
+        transform: translate(-50%, -50%);
+}
+
+.top-line{
+    font-size: 76px;
+    font-family:inherit;
+    
+} 
+
+.bottom-line{
+    font-size: 30px;
+    
+}
+
+.whats-on-spotify {
+    
+}
+
+.title p{
+    font-weight: bold;
+    font-size: 35px;
+    border-bottom:3px solid#00B172;
```

Comment:
I will admit that  I think this part was hard to get right 😁, you would actually not use a bottom border green ....

You could have done something like this for the HTML:

```html
                    <div class="fancy-border-green">
                        <h2>What’s on Spotify?</h2>
                    </div>
```

and this for the CSS:
```css
.fancy-border h2 {
    text-align: center;
}
.fancy-border-green {
	width: 360px;
	position: relative;
	margin: 0 auto;

}
.fancy-border-green:after {
    border-bottom: 4px solid #1db954;
    content: '';
    height: 1px;
    position: absolute;
    left: 0;
    bottom: -5px;
    width: 100%;
}
```